python/beeai_framework/tools/storage.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2025 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import hashlib
+import os
+import shutil
+from abc import abstractmethod
+
+
+class PythonFile:
+    def __init__(self, filename: str, id: str, python_id: str) -> None:
+        self._id = id
+        self._python_id = python_id
+        self._filename = filename
+
+    @property
+    def id(self) -> str:
+        return self._id
+
+    @id.setter
+    def id(self, value: str) -> None:
+        self._id = value
+
+    @property
+    def python_id(self) -> str:
+        return self._python_id
+
+    @python_id.setter
+    def python_id(self, value: str) -> None:
+        self._python_id = value
+
+    @property
+    def filename(self) -> str:
+        return self._filename
+
+    @filename.setter
+    def filename(self, value: str) -> None:
+        self._filename = value
+
+
+class Input:
```

Comment:
echoing @Tomas2D I believe this should be a BaseModel as well

Reply:
We don't need this class anymore. I pointed out earlier.